pyproject.toml

```
@@ -15,6 +15,22 @@ description = "Python library for working with resolution functions of neutron i
 readme = "README.md"
 license = {file = "LICENSE"}
 
+keywords = [
+    "Python",
+    "physics",
+    "inelastic neutron scattering",
+    "instrument resolution",
+]
+
+classifiers = [
+    "Development Status :: 4 - Beta",
```

There doesn't seem to be a definitive guide to these stages, but based on [some random blog](https://martin-thoma.com/software-development-stages/) I think resolution_functions is currently Alpha or pre-Alpha. Let's go with Alpha?

```suggestion
    "Development Status :: 3 - Alpha",
```